sponsorship/src/main/java/com/kyriba/conference/sponsorship/configuration/KafkaConfiguration.java

```diff
@@ -0,0 +1,48 @@
+package com.kyriba.conference.sponsorship.configuration;
+
+import com.kyriba.conference.sponsorship.domain.EmailMessage;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.kafka.core.DefaultKafkaProducerFactory;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.kafka.core.ProducerFactory;
+import org.springframework.kafka.support.serializer.JsonSerializer;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+@Configuration
+public class KafkaConfiguration
```

Comment:
There should be Kafka starter with autoconfiguration for Spring Boot

Reply:
Done